[WIP]2016-07-18-july-2016-economical-software-testing.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "July 2016 Meetup - Economical Software Testing"
+description: ""
+tags: [node.js, javascript, software testing]
```

Comment:
Change `software testing` to `testing`.

As a general note try to find an existing tag to match. You can find our existing tag collection at: http://seanode.github.io/tags

But this is totally ad hoc, so not a big deal really...